python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -952,11 +935,7 @@ def _get_event_records(
         else:
             asset_details = None
 
-        if (
-            event_records_filter.tags
-            and not self.supports_intersect
-            and self.has_table(AssetEventTagsTable.name)
-        ):
+        if event_records_filter.tags and self.has_table(AssetEventTagsTable.name):
```

Comment:
i'm a little confused about this block- we were checking for intersect support but not using it?

Reply:
oh bc it uses _apply_filter_to_query after